source/StigData/Processed/MS-Edge-1.8.xml

```diff
@@ -1429,5 +1429,75 @@ If the value for "QuicAllowed" is not set to "REG_DWORD = 0", this is a finding.
+      <DuplicateOf />
+      <Ensure>Present</Ensure>
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <Key>HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge</Key>
+      <LegacyId>
+      </LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>Verify the policy value for "Computer Configuration/Administrative Templates/Microsoft Edge/Show Hubs Sidebar" is set to "Disabled".
+
+Use the Windows Registry Editor to navigate to the following key:
+HKLM\SOFTWARE\Policies\Microsoft\Edge
+
+If the value for "HubsSidebarEnabled" is not set to "REG_DWORD = 0", this is a finding.</RawString>
+      <ValueData>0</ValueData>
+      <ValueName>HubsSidebarEnabled</ValueName>
+      <ValueType>Dword</ValueType>
+    </Rule>
+    <Rule id="V-260467" severity="medium" conversionstatus="pass" title="SRG-APP-000080" dscresource="RegistryPolicyFile">
+      <Description>&lt;VulnDiscussion&gt;Cookies must only be allowed per session and only for approved URLs as permanently stored cookies can be used for malicious intent. 
+
+Approved URLs may be allowlisted via the "CookiesAllowedForUrls" or "SaveCookiesOnExit" policy settings, but these are not requirements.&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
+      <DuplicateOf />
+      <Ensure>Present</Ensure>
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <Key>HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Edge</Key>
+      <LegacyId>
+      </LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>Verify the policy value for "Computer Configuration/Administrative Templates/Microsoft Edge/Content settings/Configure cookies" is set to "Enabled" with the option value set to "Keep cookies for the duration of the session, except ones listed in 'SaveCookiesOnExit'".
+
+Use the Windows Registry Editor to navigate to the following key:
+HKLM\SOFTWARE\Policies\Microsoft\Edge
+
+If the value for “DefaultCookiesSetting” is not set to "REG_DWORD = 4", this is a finding.</RawString>
+      <ValueData>4</ValueData>
+      <ValueName>Computer Configuration/Administrative Templates/Microsoft Edge/Content settings/Configure cookies</ValueName>
```

Comment:
The value name is still not correct - <ValueName>Computer Configuration/Administrative Templates/Microsoft Edge/Content settings/Configure cookies</ValueName>

- should be "DefaultCookieSetting" - you will need a log entry to update this or the parser needs to be update to fix this